src/components/Settings/NotificationContainerCard.tsx

```diff
@@ -138,10 +138,12 @@ const NotificationContainerCard = ({
         trailing={
           isEnable !== null && isEnable !== undefined ? (
             <Switch
-              trackColor={{
-                false: colors.border,
-                true: colors.primary,
-              }}
+              trackColor={
+                {
+                  false: theme.colors.border, true: theme.colors.primary
+                }
+              }
+              thumbColor={theme.dark ? theme.colors.text : theme.colors.background}
```

Comment:
J'te fais la suggestion de code que @Gabriel29306 demande avec une p'tite amélioration de l'indentation 😄 :


```suggestion
              trackColor={
                {
                  false: colors.border,
                  true: colors.primary
                }
              }
              thumbColor={theme.dark ? colors.text : colors.background}
```